elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -499,6 +523,16 @@ public String getActualIndexName() {
 		return actualIndexName;
 	}
 
+	@Override
+	public Analyzer createAnalyzer(String name) {
+		return new ElasticsearchAnalyzer( name );
```

Comment:
Just wanted to point out that we need a way to tell if an Analyzer is remote or if it should be locally used so an AnalyzerReference with this information would be nice.

For instance, see `ConnectedMultiFieldsPhraseQueryBuilder#createQuery`: we need a way to tell that we don't want to analyze the phrase here but pass it directly to Elasticsearch.

The need will be similar for all external search servers, Solr included.